text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/parser/TokenParser.java

```diff
@@ -364,7 +363,7 @@ private static ElementNode buildTree(
 
         case OPEN_TAG:
           final TagNode tagNode = new TagNode(node, token, message, tagProvider);
-          if (isReset(tagNode.name())) {
+          if (isReset(tagNode.name(), tagProvider)) {
```

Comment:
can we pull this in with the normal tag resolution logic at all, rather than something separate?